ingress/controllers/nginx/controller.go

```diff
@@ -747,8 +921,11 @@ func (lbc *loadBalancerController) Run() {
 	go lbc.endpController.Run(lbc.stopCh)
 	go lbc.svcController.Run(lbc.stopCh)
 
+	time.Sleep(1 * time.Second)
```

Comment:
put that in the defferring if statement instead of here? i.e something like https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/replication/replication_controller.go#L514